### PR TITLE
Packaging: add spec changes for collectd

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , @anmolbabu , Please review

**Comment by anmolbabu** [comment]
Looks good to me. Please verify if rpm installation succeeds

File: tendrl-node-agent.spec [hunk 1]
```diff
@@ -44,13 +45,22 @@ install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/node-agent
...
+install -m  0755  --directory $RPM_BUILD_ROOT%{_bindir}/config_manager
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/collectd_template
 install -Dm 0644 tendrl-node-agent.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-node-agent.service
 install -Dm 0644 tendrl-node-agent.socket $RPM_BUILD_ROOT%{_unitdir}/tendrl-node-agent.socket
 install -Dm 0644 etc/tendrl/node-agent/node-agent.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/node-agent/node-agent.conf.yaml
 install -Dm 0644 etc/tendrl/node-agent/logging.yaml.syslog.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/node-agent/node-agent_logging.yaml
 install -Dm 644 etc/tendrl/node-agent/*.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/node-agent/
 mkdir -p $RPM_BUILD_ROOT/%{_sysconfdir}/rsyslog.d
 install -Dm 644 etc/rsyslog.d/tendrl-node-agent.conf $RPM_BUILD_ROOT/%{_sysconfdir}/rsyslog.d/tendrl-node-agent.conf
+install -Dm 655 tendrl/node_agent/monitoring/collectd/commands/tendrl_monitoring_config_manager.py $RPM_BUILD_ROOT%{_bindir}/tendrl_monitoring_config_manager
```
Comment (r0h4n) [inline tendrl-node-agent.spec]: No need to copy the "tendrl_monitoring_config_manager", it is done by setup.py

File: tendrl-node-agent.spec [hunk 2]
```diff
@@ -75,8 +85,12 @@ py.test -v tendrl/node-agent/tests || :
 %dir %{_sysconfdir}/tendrl/node-agent
 %dir %{_datadir}/tendrl/node-agent
 %dir %{_sharedstatedir}/tendrl
+%attr(0655, root, root) %{_sysconfdir}/collectd_template/*
+%attr(0655, root, root) %{_libdir}/collectd/*
+
 %doc README.rst
 %license LICENSE
+%{_bindir}/config_manager
```
Comment (r0h4n) [inline tendrl-node-agent.spec]: @TimothyAsir you have copied this file as "tendrl_monitoring_config_manager" and now declaring it as "config_manager" , which is wrong

File: tendrl-node-agent.spec [hunk 3]
```diff
@@ -44,13 +45,22 @@ install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/node-agent
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/node-agent
 install -m  0755  --directory $RPM_BUILD_ROOT%{_datadir}/tendrl/node-agent
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sharedstatedir}/tendrl
+install -m  0755  --directory $RPM_BUILD_ROOT%{_libdir}/collectd/gluster/low_weight
+install -m  0755  --directory $RPM_BUILD_ROOT%{_libdir}/collectd/gluster/heavy_weight
+install -m  0755  --directory $RPM_BUILD_ROOT%{_bindir}/config_manager
```
Comment (r0h4n) [inline tendrl-node-agent.spec]: wrong name "config_manager"

Comment (TimothyAsirJeyasing) [inline tendrl-node-agent.spec]: I will remove this directory creation. By mistake i have add this dir

File: tendrl-node-agent.spec (hunk 2 above)
Comment (TimothyAsirJeyasing) [inline tendrl-node-agent.spec]: Sure, i will remove this

**Pull request merged**